app/src/main/res/values/strings.xml

```diff
@@ -25,10 +25,10 @@
     <string name="finish">Finish</string>
 
     <string name="slide_2_title">Welcome</string>
-    <string name="slide_2_desc">RethinkDNS is the easiest way to monitor network activity, bypass Internet censorship, and firewall apps on your Android device.</string>
+    <string name="slide_2_desc">RethinkDNS monitor lets you network activity, bypass Internet censorship, and firewall apps on your Android device.</string>
```

Comment:
```suggestion
    <string name="slide_2_desc">RethinkDNS lets monitor your network activity, bypass Internet censorship, and firewall apps.</string>
```

Reply:
This sentence is incorrect. Let's just stick with the current one.